### PR TITLE
Fix OpenStack playbooks

### PR DESCRIPTION
**Pull request by tomassedovic**
This shuffles a few playbooks around to fix the ordering between various
facts and prerequisites.

### CODE REVIEW

**Comment by tomassedovic** [comment]
@michaelgugino could you please take a look? This just rearranges the OpenStack playbooks to take advantage of the prerequisites, etc. Without it the openstack deployment fails.

File: playbooks/openstack/openshift-cluster/install.yml [hunk 1]
```diff
@@ -8,8 +8,8 @@
 # values here. We do it in the OSEv3 group vars. Do we need to add
 # some logic here?
 
-- name: run the cluster deploy
-  import_playbook: ../../prerequisites.yml
+- import_playbook: ../../prerequisites.yml
+- import_playbook: ../../init/cluster_facts.yml
```
Comment (michaelgugino) [inline playbooks/openstack/openshift-cluster/install.yml]: I'm unsure why this would need to be here.

Comment (tomassedovic) [reply playbooks/openstack/openshift-cluster/install.yml]: Ah you're right, that's not necessary here. I'll remove the cluster_facts.

**Comment by tomassedovic** [comment]
@michaelgugino so as it stands, the provision playbook includes `./../init/cluster_facts.yml` after the `../../prerequisites.yml` playbook. And it fails because one of the packages used in cluster_facts is only installed in the prereqs playbook later in the run.

So that's the main issue I fix here.

The rest is not installing some of the packages either because they're handled by prerequisites or because they're not necessary to run the cluster.

**Pull request merged**